components/profile_popover/profile_popover.jsx

```diff
@@ -291,11 +290,11 @@ class ProfilePopover extends React.PureComponent {
                 <div>
                     <button
                         className='user-popover__set-custom-status-btn'
-                        aria-label={Utils.localizeMessage('user_profile.custom-status.set-status', 'Set a status').toLowerCase()}
+                        aria-label={Utils.localizeMessage('user_profile.custom_status.set_status', 'Set a status').toLowerCase()}
```

Comment:
This `aria-label` repeats the message already visible on the screen.
I think this is not needed as screen readers will read the other message and we can remove this.

Refer: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

***

On a separate note, should we use `aria-labelled-by` or some other similar tags?